helpers.js

```diff
@@ -38,11 +38,17 @@ function getZone(params) {
   return zone;
 }
 
+function getAuthorizedClient(ClientClass, credentials) {
+  const clientInstance = new ClientClass({ credentials });
+  return { clientInstance, credentials };
```

Comment:
Why return credentials here? They're the exact same object passed as an argument, aren't they?

Reply:
I considered it to be AuthorizedClient, but in the end it's a bit misguiding. Fixed